src/redux/books/api/retrieveBook.js

```diff
@@ -0,0 +1,16 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import url from '../../../http-common';
+
+const RETRIVE_BOOKS = 'books/RETRIEVE_BOOK';
+
+const retrieveBooks = createAsyncThunk(RETRIVE_BOOKS, async (_, thunkAPI) => {
+  const response = await fetch(url, {
+    method: 'GET',
```

Comment:
- [ ] Nice work so far 👍🏿 on your project. I can see you are using the `fetch`  method for your API calls. But as per project requirement you are to use Axios for the API call instead of  fetch method

Reply:
well noted